modules/controller.nix

```diff
@@ -8,6 +8,14 @@ let
     mkOption
     types
   ;
+
+  # Turn it off and on again
+  steamdeck-controller-reset = pkgs.writeShellScript "steamdeck-controller-reset" ''
+    echo 0 > /sys/devices/pci0000:00/0000:00:08.1/0000:04:00.4/usb3/3-3/authorized
+    sleep 0.5
+    echo 1 > /sys/devices/pci0000:00/0000:00:08.1/0000:04:00.4/usb3/3-3/authorized
+    sleep 0.5
```

Comment:
I observed that *infrequently* steam would start and not take control of the controller.

I do not know if this actually solves the issue yet, but I haven't observed it failing since.